R/utils.R

```diff
@@ -4,37 +4,45 @@ setup_future <- function(jobs) {
     ## If running as a script enable this
     options(future.fork.enable = TRUE)
   }
-  
-  
+
+
   future::plan("multiprocess", workers = min(future::availableCores(), jobs),
                gc = TRUE, earlySignal = TRUE)
-  
-  
+  futile.logger::flog.debug("Checking the cors available - %s cores and %s jobs. using %s workers",
```

Comment:
As mentioned above. I updated the way cores are managed a little in master. A merge of these two would be really great (I think the logging here is better but ideally being able to manage cores externally is a nice feature to ha ve)